src/Util/Crypto.php

```diff
@@ -73,10 +73,11 @@ public static function rsaVerify($data, $sig, $key, $alg = 'sha256')
 	public static function meToPem($m, $e)
 	{
 		$rsa = new RSA();
-		$rsa->loadKey([
-								  'e' => new BigInteger($e, 256),
-								  'n' => new BigInteger($m, 256)
-							  ]);
+		$rsa->loadKey(
+			[
+				'e' => new BigInteger($e, 256),
+				'n' => new BigInteger($m, 256)
+			]);
```

Comment:
Please choose either the extended form
```suggestion
		$rsa->loadKey(
			[
				'e' => new BigInteger($e, 256),
				'n' => new BigInteger($m, 256)
			]
		);
```
Or the condensed form
```suggestion
		$rsa->loadKey([
			'e' => new BigInteger($e, 256),
			'n' => new BigInteger($m, 256)
		]);
```
But not a mix between both.

Reply:
hrmpf .. I had to reinstall my IDE .. Seems like there're some settings left to do ;-)